docs/technical-docs/git_and_github.md

```diff
@@ -56,6 +57,10 @@ For simplicity, the personal access token is recommended.  Authenticate by follo
 
 For details on other authentication options, check out [this resource page](https://docs.github.com/en/github/authenticating-to-github).
 
+## Notifications
+
+You can control whether GitHub sends you web notifications, e-mail notifications, or both when you are @mentioned inside GitHub. To ensure that you see these messages (especially when you are first learning GitHub), it is recommended that you select both "Email" and "Web and Mobile" under the Notifications tab of your Account Settings. In this way, when someone tags you on GitHub, you will receive an e-mail, and you will also be able to access the message from the :bell: icon in the top-right corner of the GitHub webpage.
+
 ## Basic Commands
 
 Git has several common commands that will be necessary to properly work with the the lab's repositories. These are `status`, `add`, `commit`, `push`, `pull`, and `checkout`.
```

Comment:
Is there a place where it explains the purpose of each command? Since you are mentioning it, it would make sense to add at least a short outline for each.

Reply:
I think what you're referring to exists further down the page [here](https://ndclab.github.io/wiki/docs/technical-docs/git_and_github.html#basic-commands)?